services/ui/ws/window_tree.cc

```diff
@@ -2442,13 +2442,13 @@ void WindowTree::CancelDragDrop(Id window_id) {
   wms->CancelDragDrop();
 }
 
-void WindowTree::PerformWindowMove(uint32_t change_id,
-                                   Id window_id,
-                                   ui::mojom::MoveLoopSource source,
-                                   const gfx::Point& cursor) {
+void WindowTree::PerformWindowMove(uint32_t change_id, Id window_id,
```

Comment:
suggestion: maybe it makes sense to again an early if-block for external window mode?

for example

in like 2450:

if (success && window_server_->IsInExternalWindowMode()) {
  // do our thing, and return
}

// and leave the rest of the method untouched?

Reply:
as per discussion, we modify change ids and rely on the checks perform before the call to window_server_->RunMoveLoop. so, leave the code as it is.